libraries/libfc/test/test_base64.cpp

```diff
@@ -26,21 +26,27 @@ BOOST_AUTO_TEST_CASE(base64dec) try {
    auto input = "YWJjMTIzJCYoKSc/tPUB+n5h"s;
    auto expected_output = "abc123$&()'?\xb4\xf5\x01\xfa~a"s;
 
-   BOOST_CHECK_EQUAL(expected_output, base64_decode(input));
+   std::vector<char> b64 = base64_decode(input);
+   std::string b64_str(b64.begin(), b64.end());
+   BOOST_CHECK_EQUAL(expected_output, b64_str);
```

Comment:
Again, not a required change, just a suggestion for here and the few next instances of the same pattern.
```suggestion
   BOOST_CHECK_EQUAL(std::string_view(expected_output), std::string_view(b64.begin(), b64.end()));
```